.gitignore

```diff
@@ -154,6 +154,7 @@ dist
 cache
 artifacts
 submodule-artifacts/goerli/solcInputs
```

Comment:
why do we exclude solc inputs?

Reply:
because every new deploy it has a lot of changes, that we don't want to see in PR